antora.yml

```diff
@@ -1,13 +1,13 @@
 name: operations-manual
 title: Operations Manual
-version: '5'
+version: '2025.01'
 start_page: ROOT:index.adoc
 nav:
 - modules/ROOT/content-nav.adoc
 asciidoc:
   attributes:
     neo4j-version: '5'
```

Comment:
Should the `neo4j-version` be '2025.01' not '5'?

Reply:
I think I've fixed that.